internal/app/machined/internal/sequencer/v1alpha1/v1alpha1_sequencer.go

```diff
@@ -122,10 +124,26 @@ func (d *Sequencer) Boot() error {
 			"setup /var",
 			rootfs.NewVarDirectoriesTask(),
 		),
+		phase.NewPhase(
+			"unmount /boot",
+			rootfs.NewUnmountSystemDisksTask(constants.BootPartitionLabel),
+		),
+		phase.NewPhase(
+			"mount /boot as rw",
+			rootfs.NewMountSystemDisksTask(constants.BootPartitionLabel),
+		),
```

Comment:
Should we maybe mount as R/W to begin with earlier in boot? Then we won't have to do these.

Reply:
Andrey's concern around that was that because it's a FAT file system there's a higher chance of corruption. So we should mount as rw only when we needed it.

Reply:
my concern was close to that, but basically not to leave it r/w longer than we need, as if we screw `/boot`, we screw whole node